client/src/styles/global.scss

```diff
@@ -1,2 +1,31 @@
-@import "~@trussworks/react-uswds/lib/uswds.css";
 @import "~@trussworks/react-uswds/lib/index.css";
+@import "~@trussworks/react-uswds/lib/uswds.css";
+
+.j40FloatLogoLeft {
+  width: 100%;
+  padding-left: 50%;
+}
+
+.j40Footer {
+  font-family: "sans-serif";
+  font-size: 12px;
+  line-height: 24px;
```

Comment:
We should avoid manual font-sizes and use [font size classes](https://designsystem.digital.gov/utilities/font-size-and-family/) instead - these will work in accordance with the overall site relative font expectations